tests/data_test.py

```diff
@@ -30,9 +30,13 @@ def test_datasets(ds_cls: type[VisionDataset]) -> None:
 
 
 def test_audio_dataset() -> None:
-    """Tests basic functionality of an audio dataset."""
-    dataset = PBAudioDataset()
+    """Tests basic functionality of an audio dataset base class."""
+    x = torch.rand(1, 10)
+    audio_dir = Path(r"Sample path")
+    dataset = PBAudioDataset(x=x, audio_dir=audio_dir)
+
     assert dataset is not None
+    assert len(dataset) == len(x)
 
```

Comment:
Maybe something like
```suggestion
    assert str(dataset).splitlines()[0] == "Dataset PBAudioDataset"
    assert str(dataset).splitlines()[1].strip() == "Number of datapoints: 1"
```
to test the repr method

Reply:
Good addition, I hadn't thought to test the repr.